src/main/java/com/google/devtools/build/lib/authandtls/AuthAndTLSOptions.java

```diff
@@ -139,6 +139,41 @@ public class AuthAndTLSOptions extends OptionsBase {
               + "pings are disabled, then this setting is ignored.")
   public Duration grpcKeepaliveTimeout;
 
+  @Option(
+      name = "experimental_experimental_credential_helper",
```

Comment:
Remove the extra `experimental`.

Reply:
Of course I copy&pasted to the wrong flag 🤦‍♀️